patches/nginx-ssl-cert.patch

```diff
@@ -0,0 +1,37 @@
+diff --exclude '*~' '--exclude=*.swp' -upr a/src/event/ngx_event_openssl.c b/src/event/ngx_event_openssl.c
+--- a/src/event/ngx_event_openssl.c	2014-08-05 04:13:07.000000000 -0700
++++ b/src/event/ngx_event_openssl.c	2014-09-12 12:17:33.034582693 -0700
+@@ -1121,6 +1121,21 @@ ngx_ssl_handshake(ngx_connection_t *c)
+         return NGX_AGAIN;
+     }
+ 
++    if (sslerr == SSL_ERROR_WANT_X509_LOOKUP) {
++        c->read->handler = ngx_ssl_handshake_handler;
++        c->write->handler = ngx_ssl_handshake_handler;
++
++        if (ngx_handle_read_event(c->read, 0) != NGX_OK) {
++            return NGX_ERROR;
++        }
++
++        if (ngx_handle_write_event(c->write, 0) != NGX_OK) {
++            return NGX_ERROR;
++        }
++
++        return NGX_AGAIN;
++    }
++
+     err = (sslerr == SSL_ERROR_SYSCALL) ? ngx_errno : 0;
+ 
+     c->ssl->no_wait_shutdown = 1;
+diff --exclude '*~' '--exclude=*.swp' -upr a/src/event/ngx_event_openssl.h b/src/event/ngx_event_openssl.h
+--- a/src/event/ngx_event_openssl.h	2014-08-05 04:13:07.000000000 -0700
++++ b/src/event/ngx_event_openssl.h	2014-09-12 12:16:32.016208272 -0700
+@@ -56,6 +56,8 @@ typedef struct {
+     ngx_event_handler_pt        saved_read_handler;
+     ngx_event_handler_pt        saved_write_handler;
+ 
++    void                       *lua_ctx;  /* used by 3rd-party modules */
```

Comment:
@lziest Oh, this patch is already obsolete. I've just removed the whole patch file :) Thanks for the catch.